views/accounts/weekly-report.pug

```diff
@@ -0,0 +1,18 @@
+.weekly-summary-modal
```

Comment:
Please break this file up with appropriate whitespace (perhaps by detail section?)